inhouse_bot/cogs/admin_cog.py

```diff
@@ -32,6 +38,9 @@ class AdminCog(commands.Cog, name="Admin"):
     def __init__(self, bot: InhouseBot):
         self.bot = bot
 
+        # if INHOUSE_BOT_TOURNAMENTS:
```

Comment:
what does this do exactly?

Reply:
I added an extra comment to explain why I left that there. QueueCog has a similar block commented out. 

At first I had it there to disable the !won command when in TOURNAMENTS mode, because the games are scored automatically. But in the case there's a failure somehow, I'm leaving it in case players need to score so they aren't stuck.